src/plugins/intel_cpu/src/nodes/executors/acl/acl_utils.cpp

```diff
@@ -5,9 +5,60 @@
 #include "acl_utils.hpp"
 #include "support/Mutex.h"
 
-void ov::intel_cpu::configureThreadSafe(const std::function<void(void)>& config) {
+namespace ov {
+namespace intel_cpu {
+
+void configureThreadSafe(const std::function<void(void)>& config) {
     // Issue: CVS-123514
     static arm_compute::Mutex mtx_config;
     arm_compute::lock_guard<arm_compute::Mutex> _lock{mtx_config};
     config();
 }
+
+bool getActivationLayerInfo(Algorithm algorithm,
```

Comment:
`getActivationLayerInfo` function returns boolean value and updates `arm_compute::ActivationLayerInfo` instance.
Separate to two different functions, please:
* `getActivationLayerInfo` function has to return `arm_compute::ActivationLayerInfo` instance
* `checkActivationLayerInfo` should check element-wise operation type.

Reply:
@eshoguli corrected